ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -686,7 +684,7 @@ static const func_info_t func_infos[] = {
 	F1("mb_strtolower",                         MAY_BE_FALSE | MAY_BE_STRING),
 	F1("mb_language",                           MAY_BE_FALSE | MAY_BE_TRUE | MAY_BE_STRING),
 	F1("mb_internal_encoding",                  MAY_BE_FALSE | MAY_BE_TRUE | MAY_BE_STRING),
-	F1("mb_http_input",                         MAY_BE_FALSE | MAY_BE_STRING),
+	F1("mb_http_input",                         MAY_BE_FALSE | MAY_BE_STRING| MAY_BE_ARRAY),
```

Comment:
Does the `MAY_BE_ARRAY` need any more specification? If so, I'll try to figure it out tomorrow how the actual structure looks like.

Reply:
Good catch, it does indeed! MAY_BE_ARRAY by itself only allows empty arrays.